packages/amplify-graphql-api-construct-tests/src/__tests__/backends/amplify-table/rate-limit/updateTableTtl/disabled/app.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import { App, Stack, Duration } from 'aws-cdk-lib';
+// @ts-ignore
+import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/graphql-api-construct';
+
+const packageJson = require('../package.json');
+
+const app = new App();
+const stack = new Stack(app, packageJson.name.replace(/_/g, '-'), {
+  env: { region: process.env.CLI_REGION || 'us-west-2' },
+});
+const schema =
+  `input AMPLIFY { globalAuthRule: AuthRule = { allow: public } }\n` +
+  Array.from({ length: 60 }, (_, i) => i + 1)
```

Comment:
There is a hard limit for `2500` resources for nested stack within one CFN operation (CRUD). For one nested stack of a model type, there are ~40 resources, which indicates the test for `60` is reasonable. In the real use case, this number will be smaller as there will be additional resources in the nested stack(such as function)
<img width="734" alt="image" src="https://github.com/aws-amplify/amplify-category-api/assets/20614187/b84bfca4-c996-491c-90eb-d2b558f0092e">